vimeo-networking/src/test/java/com/vimeo/networking/model/HlsVideoFileTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.model;
+
+import com.vimeo.networking.Utils;
+
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link HlsVideoFile}.
+ * <p>
+ * Created by zetterstromk on 8/3/17.
+ */
+public class HlsVideoFileTest {
+
+    @Test
+    public void verifyTypeAdapterWasGenerated() throws Exception {
+        Utils.verifyTypeAdapterGeneration(HlsVideoFile.class);
```

Comment:
FWIW, I see both sides of this. I guess it could get annoying to follow this pattern in networking when we will have a ton of model classes; however, Brent's point about the sub-packages makes me lean toward keeping this as-is. Otherwise, we'd probably end up with inconsistencies in our tests for models as some of them would warrant breaking out into dedicated classes and some wouldn't and it could be hard to find all of the relevant tests.